dhcp/install.sh

```diff
@@ -12,7 +12,11 @@ INTERNET=$(ip route | grep default | cut -d' ' -f5)
 LOCAL=$(ip route | grep -v default | cut -d' ' -f3 | grep -v $INTERNET | head -1)
 
 # configuration
-echo "dhcp-range=${NETWORK}.50,${NETWORK}.150,12h" >> /etc/dnsmasq.conf 
+cat <<EOT >> /etc/dnsmasq.conf
+enable-tftp
+pxe-service=x86PC, "Install Linux", /ltsp/amd64/pxelinux, $LAN_IP
```

Comment:
Where is LAN_IP defined?

Reply:
It is defined in `settings.sh`
I have set source to `/vagrant/settings.sh`
and its folder synced with vagrant.

Reply:
Can't we merge this Vagrantfile with the one at root level and use "standalone" flag to make respective changes.

Reply:
For improvement in future we can have single Vagrantfile to support standalone & non-standalone mode.

Reply:
good idea.